src/packages/loader/index.js

```diff
@@ -1,4 +1,5 @@
 import fs from '../fs';
```

Comment:
I'm nitpicking but can we add an export from the main fs module (./packages/fs/index.js) like this:

```javascript
export default fs;
export isJSFile from './utils/is-js-file';
```

And then import in this file like:

```javascript
import fs, { isJSFile } from '../fs';
```